alpha/lib/model/LiveEntry.php

```diff
@@ -629,80 +642,80 @@ public function setMediaServer($index, $hostname, $applicationName = null)
 
 		KalturaLog::debug("about to setMediaServer. index: $index, hostname: $hostname");
 		$this->setLastBroadcast(kApiCache::getTime());
-		$server = new kLiveMediaServer($index, $hostname, $mediaServerNode->getDc(), $mediaServerNode->getId(), $applicationName ? $applicationName : $mediaServerNode->getApplicationName());
-		$this->putInCustomData("server-$index", $server, LiveEntry::CUSTOM_DATA_NAMESPACE_MEDIA_SERVERS);
-		$this->setLiveStatus(LiveEntryStatus::PLAYABLE, $index);
+		$dbLiveEntryServerNode->setStatus(EntryServerNodeStatus::PLAYABLE);
+		$dbLiveEntryServerNode->setServerType($index);
+		$dbLiveEntryServerNode->setServerNodeId($mediaServerNode->getId());
+		$dbLiveEntryServerNode->save();
+		$this->setLiveStatus(EntryServerNodeStatus::PLAYABLE, $index);
```

Comment:
I see that save on the entry is no longer called from the KalturaLiveEntryService so or add it after here after the setLiveStatus call or in KalturaLiveEntryService function setLiveStatusOrMediaServer inside the try block